### ISSUE TITLE
Discussion: Tracking progress and training completion 

### ISSUE THREAD

**Issue by unshakenme** [unshakenme/OSPAT#7]
One day this may become a legitimate  resource for therapists seeking a free, self led training option. 

What open source solutions exist that would allow us to turn this into a training program that can track progress and automate a certificate of completion?

**Comment by CheekeyMonkey** [comment]
There are various open-source LMS platforms that can be used. I have experience with building on lots of different platforms, however, content is king! Lets make this the best we can and then maybe we will come to a point where we could start publishing it in a system somewhere... :)

**Comment by unshakenme** [comment]
Excellent and agree. This is more for future reference and my own learning. Would those systems allow for pulling directly from GitHub? I’m not a developer so am learning as I go.

**Comment by CheekeyMonkey** [comment]
No, it would not pull directly. The best way to do it is as you have done. Make a repository, refine it, get more people involved working on it, and then when it's at a stage of acceptable quality, publish it as V1 of the course... I think, however, the content thus far, although good, is not sufficient.

**Comment by voyagermedical** [comment]
Yes, I agree it needs to have more structured content. May I propose...

**1. Introduction**
General introduction to the course.
Take from Read me?

**1.2. History**
Indigenous world wide use,  avoid woo?! - kykeon, unnamed religion, Eleusinian mysteries? etc? Or start with Wasson? Huxley? Perhaps use images, with permission from: https://www.facebook.com/groups/psychedelicsynchronicity

**1.3. Politics**
How the legality got there, including _the war on drugs_ , Tim Leary, criminalisation, psychedelic renaissance, legalisations and pending legalisations.

**2. Biology** 
3.1 Origins
Why do these compounds exist? Plant / funghi chemical warfare...Insecticide? **No woo.**

**2. Chemistry** 
2.1 Chemical classes: phenyletyhalmines / tryptamins / deliriants. Explain why they are in these classes.
2.2 Individual medicines: Tryptamines

Psilocybin / psilocin found in psilocybin mushrooms
[DMT](https://burnzero.com/DMT) found in acacias and ayahuasca
5-MeO-DMT found in toad venom
Ergoline

LSD derived from ergot
Phenylethylamines

Mescaline found in the peyote cactus
MDMA synthetically made
2C-B synthetically made
Deliriants

Muscimol
Ibogane
Dissociatives

Ketamine synthetically made

**4. Medicine**
4.1. Effects
4.2. Psychedelic Comparison

**5. Therapy**
5.1 Summarize best practice.

**Comment by unshakenme** [comment]
I like where you are going with this structure, but I don't think the average therapist is going to care or need to know much depth on the biology/chemistry. Those would go better in the elective category rather than the core curriculum.

This topic should have it's own Issue thread...

**Comment by voyagermedical** [comment]
So concentrate on therapeutics as this is the core audience, fine.

More like this?

1. Introduction
General introduction to the course.
Take from Read me?

1.2. History
Indigenous world wide use, avoid woo?! - kykeon, unnamed religion, Eleusinian mysteries? etc? Or start with Wasson? Huxley? Perhaps use images, with permission from: https://www.facebook.com/groups/psychedelicsynchronicity

1.3. Politics
How the legality got there, including the war on drugs , Tim Leary, criminalisation, psychedelic renaissance, legalisations and pending legalisations.

2. Therapeutics

2.1 Medicines

2.1.2Tryptamines
Psilocybin / psilocin found in psilocybin mushrooms
[DMT] found in acacias and ayahuasca
5-MeO-DMT found in toad venom
Ergoline
LSD derived from ergot

2.1.3 Phenylethylamines
Mescaline found in the peyote cactus
MDMA synthetically made
2C-B synthetically made
2.1.4 Deliriants
Muscimol
Ibogane

2.1.5 Dissociatives
Ketamine synthetically made

2.2. Psychedelic Comparison

3. Therapy
3.1 Summarize best practice.
